src/components/Title.jsx

```diff
@@ -1,7 +1,23 @@
 import React from "react";
 
 const Title = () => {
-  return <div>Title</div>;
+  const containerHeight = "170px";
+
+  return (
+    <div
+      className="relative w-2/3 overflow-hidden"
+      style={{ height: containerHeight }}
+    >
+      <img
+        src="/svg/Landing.svg"
+        alt="Landing"
+        className="absolute bottom-0 left-0 w-full h-auto"
+      />
+      <div className="absolute bottom-0 left-0 right-0 flex justify-center items-center">
+        <div>Title</div>
```

Comment:
Display your text prop here `{text}`

Reply:
<img width="1719" alt="image" src="https://github.com/acm-ucr/saf-website/assets/156125299/3ac3a5c7-c2c4-4744-8ff3-17fd3f677cc3">
Not sure what font to use but this was similar